tests/test_indexing.py

```diff
@@ -27,4 +27,14 @@ def test_warc_ipwbIndexerBrokenWARCRecord():
     assert ipwbTest.countCDXJEntries(cdxj) == 1
 
 
+# WARC/1.1 allows for dates of length that are not easily converted
+# to 14-digits. This test highlights the failures from a WARC
+# exhibiting these dates.
+@pytest.mark.ipwbIndexerVariableSizedDates
+def test_warc_ipwbIndexerVariableSizedDates():
+    pathOfBrokenWARC = os.path.join(
+        os.path.dirname(__file__) +
+        '/../samples/warcs/variableSizedDates.warc')
```

Comment:
When using `os.path.join`, each path component should be passed as individual parameters, so that the function uses an OS-dependent path separator.

Reply:
This was copied from other tests. If we revise a means, we should do so across the board. I agree with your suggestion but currently tests are very sparse.

Reply:
You may want to take this opportunity to correct it.

Reply:
Do you know of a standard approach at specifying this? I can imagine an assortment of ways. One in particular is to read the delimiter and `delimited.join()`.

Reply:
I have added a code suggestion in a separate comment to tackle this.